README.md

```diff
@@ -332,6 +332,29 @@ make test-e2e
 ```
 These e2e tests would be performed by creating a `kind` cluster locally
 
+### Roadmap
+
+High level overview of the main priorities for 2024:
+
+- Allocate MIG slices on Nvidia GPUs on demand
+- Configire allocated slices on GPUs and bind containers to slices
+- Release and unconfigure slices when pods are completed or deleted
+- Ability to graceful termination of slices
+- Account ofr node classical resources when selecting a node
+- Schedule pods in average of 10 seconds when resources are available
+- Kubernetes quota system integration
+- Konflux onboarding
+- Operator SDK integration
+
+Future tasks:
+- Stable integration with project Kueue
```

Comment:
Should it be in the `Future tasks` or current roadmap?

cc @kannon92 @mrunalp

Reply:
I think future is fine for now. I'm not sure Kueue and Instaslice will be ready on the first release of this.

Reply:
alright, thanks.